atd-etl/cris_import/cris_import.py

```diff
@@ -0,0 +1,801 @@
+        "bastion_host": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Database Bastion",
+            "opvault": VAULT_ID,
+        },
+        "bastion_ssh_username": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Bastion ssh Username",
+            "opvault": VAULT_ID,
+        },
+        "database_host": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Database Host",
+            "opvault": VAULT_ID,
+        },
+        "database_username": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Database Username",
+            "opvault": VAULT_ID,
+        },
+        "database_password": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Database Password",
+            "opvault": VAULT_ID,
+        },
+        "database_name": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Database Name",
+            "opvault": VAULT_ID,
+        },
+        "database_ssl_policy": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Database SSL Policy",
+            "opvault": VAULT_ID,
+        },
+        "aws_access_key": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS Access key",
+            "opvault": VAULT_ID,
+        },
+        "aws_secret_key": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS Secret key",
+            "opvault": VAULT_ID,
+        },
+        "s3_archive_bucket_name": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.S3 Archive Bucket Name",
+            "opvault": VAULT_ID,
+        },
+        "s3_archive_path": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.S3 Archive Path",
+            "opvault": VAULT_ID,
+        },
+        "graphql_endpoint": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.GraphQL Endpoint",
+            "opvault": VAULT_ID,
+        },
+        "graphql_endpoint_key": {
+            "opitem": "Vision Zero CRIS Import",
+            "opfield": f"{DEPLOYMENT_ENVIRONMENT}.GraphQL Endpoint key",
+            "opvault": VAULT_ID,
+        },
+        "sftp_endpoint_private_key": {
+            "opitem": "SFTP Endpoint Key",
+            "opfield": ".private key",
+            "opvault": VAULT_ID,
+            },
+    }
+
+    # instantiate a 1Password client
+    client: Client = new_client(ONEPASSWORD_CONNECT_HOST, ONEPASSWORD_CONNECT_TOKEN)
+    # get the requested secrets from 1Password
+    return onepasswordconnectsdk.load_dict(client, REQUIRED_SECRETS)
```

Comment:
These lines, 1-190 are all worth giving a close look at. It's a whole new `main()` function and a `get_secrets()` function.